src/components/elementView/elementTab.tsx

```diff
@@ -183,7 +183,7 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
       ? API_ROOT + "api/sites/datacenters/pages"
       : API_ROOT + "api/" + path + "/pages";
     return axios
-      .post(API_ROOT + url, { filters: filtersCopy }, config)
+      .post(url, { filters: filtersCopy }, config)
```

Comment:
technicallllyyy would be cleaner + more consistent with how we do it else where to leave API_ROOT + url, but remove API_ROOT from the the variable url itself since that is always going to be there